Makes untrusted domain error on info

AndyXheli opened the pull request:
Signed-off-by: Andy Xheli <axheli@axtsolutions.com>

Since https://github.com/nextcloud/server/commit/e6d9ef2e38daffcab808eaa41b18ab16c6253b97 was applied logs get filled up with Trusted domain error. "X.X.X.X tried to access using "X.X.X.X" as host alot of users missed important errors do to https://github.com/nextcloud/server/commit/e6d9ef2e38daffcab808eaa41b18ab16c6253b97   please see https://github.com/nextcloud/server/issues/32599

This should fix. 
https://github.com/nextcloud/server/issues/32599#event-7281164903

Commit:
Makes untrusted domain error on info

Signed-off-by: Andy Xheli <axheli@axtsolutions.com>

Since https://github.com/nextcloud/server/commit/e6d9ef2e38daffcab808eaa41b18ab16c6253b97 was applied logs get filled up with Trusted domain error. "X.X.X.X tried to access using "X.X.X.X" as host alot of users missed important errors do tohttps://github.com/nextcloud/server/commit/e6d9ef2e38daffcab808eaa41b18ab16c6253b97   please see https://github.com/nextcloud/server/issues/32599

This should fix. 
https://github.com/nextcloud/server/issues/32599#event-7281164903


Signed-off-by: Andy Xheli <axheli@axtsolutions.com>

BJKle commented:
How ist this working?
Is it just luck if the reviewers pick this and check it or is there a formal process available and hasn't been triggered yet?
It is just one word, which is reverted to the value before version 24.
It would be great to have it back with version 25 :-)
Thanks

andyxheli commented:
Hi All. 

Can you please get a review on this?

szaimen reviewed (approved):
fine by me

come-nc reviewed (approved):
Not clear if there is not a better solution here, but for the time being…

andyxheli commented:
Thank you!! I'm sure there will be some happy admins :)

andyxheli commented:
@szaimen is this also backported to NC 24 and NC 25 ?

andyxheli commented:
Hey @szaimen I just updated to NC 24.0.6 and im not seeing this change in there. and i updated my NC 25 to RC3 and i dont see the change there also Can you please look into this ?

szaimen commented:
/backport to stable25

szaimen commented:
/backport to stable24
